app/controllers/application_controller/filter.rb

```diff
@@ -1627,9 +1627,9 @@ def build_listnav_search_list(db)
     temp = MiqSearch.new
     temp.description = _("ALL")
     temp.id = 0
-    @def_searches = MiqSearch.where(:db => db).visible_to_all.sort_by { |s| s.description.downcase }
+    @def_searches = MiqSearch.where(:db => [db, db.constantize.to_s]).visible_to_all.sort_by { |s| s.description.downcase }
```

Comment:
we were talking with @himdel if this should fail on non existent class. It probably should. :-) Or not?